app-scroll-effects/app-scroll-effects-behavior.html

```diff
@@ -285,7 +285,33 @@
   }];
 
   /**
-   * Defines the namespace for the effects
+   * Defines the namespace for the scroll effects.
+   *
+   * The `Polymer.AppScrollEffects` is a helper function that scrolls to a particular 
+   * set of coordinates in a scroll target without scroll effects transitions.
+   *
+   * @param {number} scrollTop 
+   * @param {?HTMLElement} scrollTarget
    */
-  Polymer.AppScrollEffects = Polymer.AppScrollEffects || {};
+  Polymer.AppScrollEffects = function scroll(scrollTop, scrollTarget) {
+    var docEl = document.documentElement;
+
+    docEl.setAttribute('disabled-scroll-effects', '');
+
+    if (!(scrollTarget instanceof HTMLElement) || scrollTarget === docEl) {
+     // the document element
+      window.scrollTo(0, scrollTop);
+
+    } else if (scrollTarget.scroll) {
+      // if the scroll target has a scrolling mechanism
+      scrollTarget.scroll(0, scrollTop);
```

Comment:
Why do we need to support this custom `scroll` mechanism?  There is no contract what this `scroll` API would be like and here we are assuming it has certain arguments.